README.md

```diff
@@ -12,6 +12,14 @@
 > **Warning**
 > Quantum requires react@16.14.0 or higher.
 
+## Notices ⚠️
+
+For performance and long-term maintenance reasons, we do not recommend using [MUI](https://mui.com/material-ui/getting-started/) in Quantum component implementations. MUI is included in our library only to enable the use of [Material Icons](https://mui.com/material-ui/material-icons/).
```

Comment:
For performance and long-term maintenance reasons, we do not recommend using [MUI](https://mui.com/material-ui/getting-started/) in Quantum components implementation. MUI is present in our library only to allow the use of [Material Icons](https://mui.com/material-ui/material-icons/).